core/src/main/scala/cats/Show.scala

```diff
@@ -25,6 +25,13 @@ object Show {
     def show(a: A): String = a.toString
   }
 
+  def doShow[A](a: A)(implicit ev: Show[A]): String =
```

Comment:
I'm not crazy about "doShow". Sorry for the bike shed. Can we use callShow, showString, showOf or something? "do" sounds like an effect to me (maybe because of Haskell).